index.html

```diff
@@ -15,26 +15,201 @@
 </head>
 
 <body>
-    <nav>
-      <i class='bx bx-menu bx-sm'></i>
-    </nav>
+  <nav>
+    <i class='bx bx-menu bx-sm'></i>
+  </nav>
   <section class="headline">
     <div class="container">
       <h1>Hey there. <br>
         I’m Tsige.</h1>
-      <h2 class="green">I’m a software developer! </h2></div>
-      <p>I can help you build any software as your need
-        including DesktopApp ,Mobile feature or website.
-        For more Please Look through
-        some of my work and experience that I have!</p><br>
-      <ul>
-        <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
-        <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
-        <li><a href=""><img src="icons/medium.svg"></i></a></li>
-        <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
-        <li><a href=""><img src="icons/angellist.svg"></a></li>
-      </ul>
+      <h2 class="green">I’m a software developer! </h2>
     </div>
-      </section>
+    <p>I can help you build any software as your need
+      including DesktopApp ,Mobile feature or website.
+      For more Please Look through
+      some of my work and experience that I have!</p><br>
+    <div><ul>
+      <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
+      <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
+      <li><a href=""><img src="icons/medium.svg"></i></a></li>
+      <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
+      <li><a href=""><img src="icons/angellist.svg"></a></li>
```

Comment:
[OPTIONAL] Please stick to the Figma design your social media icons are not the same as the Figma design so kindly fix this issue :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-03 14-00-12](https://github.com/tsigewossne/portfolio/assets/45226848/582986f1-34ce-4685-999e-1e481d584b6f)

**Figma Design**
![Screenshot from 2023-06-03 14-01-16](https://github.com/tsigewossne/portfolio/assets/45226848/5e8f0c1d-0cc9-49e3-8d4f-51297617f7ec)